AEPCore/Sources/core/MobileCore.swift

```diff
@@ -38,6 +39,40 @@ public final class MobileCore: NSObject {
     /// Pending extensions to be registered for legacy support
     static var pendingExtensions = ThreadSafeArray<Extension.Type>(identifier: "com.adobe.pendingExtensions.queue")
 
+    public static func start(with appID: String,
+                             logLevel: LogLevel? = nil,
+                             configUpdates: [String: Any]? = nil,
+                             additionalContextData: [String: Any]? = nil,
+                             disableLifecycleStart: Bool? = nil,
+                             applicationState: UIApplication.State? = nil
+                             ) {
+        if let logLevel = logLevel {
+            setLogLevel(logLevel)
+        }
+        // Register Extensions, call configureWithAppId from callback
+        let classList = ClassFinder.classes(conformToProtocol: Extension.self)
+        let filteredClassList = classList.filter { $0 !== AEPCore.EventHubPlaceholderExtension.self && $0 !== AEPCore.Configuration.self }.compactMap { $0 as? NSObject.Type }
+        registerExtensions(filteredClassList) {
+            configureWith(appId: appID)
+            if let configUpdates = configUpdates {
+                updateConfigurationWith(configDict: configUpdates)
+            }
+            
+            // If disableLifecycleStart flag is non nil and true, set lifecycle notification listeners
+            guard let disableLifecycleStart = disableLifecycleStart, disableLifecycleStart == true else {
```

Comment:
This variable name is misleading. Can we name it `automaticLifecycleTracking` or something similar?